app/components/tabpanel_component.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class TabpanelComponent < ViewComponent::Base
+  renders_one :body
+  renders_one :subheading
+
+  def initialize(id:, title:, hidden: false, data: {})
+    @id = id
+    @hidden = hidden
+    @title = title
+    @data = data
+  end
+
+  attr_reader :id, :title, :data
+
+  def hidden?
+    @hidden
+  end
```

Comment:
Is there a reason hidden isn't added the attr_reader?

Reply:
@dnoneill because we want a method ending in a question mark.

Reply:
Why? Also can we use an alias if we really want the method to end in a question mark.

Reply:
It's ideomatic ruby to use methods ending with a question mark to indicate boolean return value.  I don't want to have a method defined (by attr_accessor) that we don't need.

Reply:
ok